packages/web-pkg/src/components/SideBar/Files/FileInfo.vue

```diff
@@ -42,12 +43,21 @@ export default defineComponent({
   },
   setup() {
     const resourcesStore = useResourcesStore()
+    const { $gettext } = useGettext()
 
     const resource = inject<Resource>('resource')
+    const space = inject<SpaceResource>('space')
     const areFileExtensionsShown = computed(() => resourcesStore.areFileExtensionsShown)
 
+    const resourceName = computed(() => {
+      return isPersonalSpaceResource(unref(space)) && unref(resource).path === '/'
+        ? $gettext('Personal')
```

Comment:
Possible to use the name of the space in that case? Because we're already setting `Personal` as space name for the personal space if I remember correctly.

Reply:
Makes it a little bit less readable IHMO, but yes